.openshift-ci/debug-tips.md

```diff
@@ -0,0 +1,21 @@
+# Debug OpenShift CI jobs
+
+## Download all artifacts
+
+1. Visit the jobs page
+2. Click on the `Aritfacts` link
+3. Append `artifacts/merge-gke-nongroovy-e2e-tests/stackrox-stackrox-e2e-test/artifacts/howto-locate-other-artifacts-summary.html` to the URL openend in your browser.
```

Comment:
There are two sources of 'Artifacts'. The Openshift CI (openshift/release) ones and the StackRox ones. The former are found through the top right Artifacts links. The latter are found through the "Additional StackRox e2e artifacts" pane in the middle of a job view [e.g.](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/stackrox_stackrox/10768/pull-ci-stackrox-stackrox-master-gke-qa-e2e-tests/1790718430115008512). That pane can be clicked to get a gsutil command to copy artifacts locally. The  "Additional StackRox e2e artifacts"  are typically where central DB dumps are found, so that may be the one we want to highlight but they are only accessible for RH engineers.

Reply:
I've resolved this conversation, if you have feedback happy to add it later @gavin-stackrox .